ci.sh

```diff
@@ -30,7 +30,7 @@ sudo mv /tmp/os-test/openshift-tests /usr/local/bin/
 
 # Run createdisk script
 export CRC_ZSTD_EXTRA_FLAGS="-10 --long"
-./createdisk.sh crc-tmp-install-data
+SNC_GENERATE_MACOS_BUNDLE=0 ./createdisk.sh crc-tmp-install-data
```

Comment:
@cfergeau I am thinking around https://github.com/crc-org/snc/pull/726#discussion_r1205310183 one if we can stop using internal repo if our CI images are migrated to RHEL-9

Reply:
So I am going ahead with making change in the `ci_microshift.sh` script also. Since our CI nodes are still not on RHEL-9, which might take some time. Once we move the CI nodes to RHEL-9 then might be reconsider it.